doc/api/cluster.md

```diff
@@ -6,9 +6,11 @@
 
 <!-- source_link=lib/cluster.js -->
 
-A single instance of Node.js runs in a single thread. To take advantage of
-multi-core systems, the user will sometimes want to launch a cluster of Node.js
-processes to handle the load.
+Clusters of Node.js processes can be used to run multiple instances of Node.js
+that can distribute workloads among their application threads. When process
+isolation is not needed, the newer [worker threads][] module can be used
```

Comment:
I wasn't sure about this. The worker docs are inconsistent in how the spell the name. How about this?

```suggestion
isolation is not needed, the newer [`worker_threads`][] module can be used
```

Reply:
Titles use sentence case: _Worker threads_

But otherwise, in ordinary text, I think _worker threads_ is the way to go. The Microsoft guide (that we've adopted for our style) says: ["When in doubt, don't capitalize"](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize)

All that said `worker_threads` is a fine way to avoid the whole question!

Reply:
Thanks for the clarification @Trott! :)